client/blocks/reader-post-card/photo.jsx

```diff
@@ -66,30 +66,40 @@ class PostPhoto extends React.Component {
 			backgroundRepeat: 'no-repeat',
 			backgroundPosition: 'center'
 		};
-
+		let newWidth, newHeight;
 		if ( this.state.isExpanded ) {
 			const viewportHeight = this.getViewportHeight();
 			const cardWidth = this.state.cardWidth;
 			const { width: naturalWidth, height: naturalHeight } = imageSize;
 
-			const newHeight = Math.min(
+			newHeight = Math.min(
 				( naturalHeight / naturalWidth ) * cardWidth,
 				viewportHeight - 176
 			);
-
+			newWidth = ( naturalWidth / naturalHeight ) * newHeight;
 			featuredImageStyle.height = newHeight;
+			featuredImageStyle.width = newWidth;
 		}
 
 		const classes = classnames( {
 			'reader-post-card__photo': true,
 			'is-expanded': this.state.isExpanded
 		} );
 
+		const divStyle = this.state.isExpanded
+			? { height: newHeight, width: newWidth, margin: 'auto' }
+			: {};
+
 		return (
-			<a className={ classes } href={ href } style={ featuredImageStyle } onClick={ this.handleClick }>
-				<div ref={ this.handleWidthDivLoaded } style={ { width: '100%' } }></div>
-				{ children }
-			</a>
+			<div style={ divStyle }>
+				<a className={ classes } href={ href } style={ featuredImageStyle } onClick={ this.handleClick }>
+					<div ref={ this.handleWidthDivLoaded } style={ { width: '100%' } }></div>
+					{ children }
+				</a>
+				<h1 className="reader-post-card__title">
+					<a className="reader-post-card__title-link" href={ this.props.href }>{ this.props.title }</a>
```

Comment:
> maybe use a new class here and on the h1? that way we can kill the override and just style it directly.

what do you mean kill the override?  Right now I'm only explicitly inlining style for pieces that need to calculated via js (width/heights).

> any idea how moving the h1 and link around affects screen readers? This would change the normal flow of the document.

We aren't moving the h1 around because once its deemed a photo post it stays put!
But I'll confirm that this works in chrome tablet mode 👍